### PR TITLE
deadly skils

### PR DESCRIPTION
**Pull request by emcek**

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 introduction of emcek aka mipl
-
-My fav path and code schole is JavaScript and Electives.
+deadly skills
+==============
+* Git
+* JavaScript
+* Ruby
```
Comment (codeschool-kiddo) [inline introduction.md:6]: Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

**Comment by emcek** [comment]
looks good

**Comment by emcek** [comment]
@codeschool-kiddo look good

File: introduction.md (hunk 1 above)
Comment (emcek) [inline introduction.md:6]: @codeschool-kiddo looks good

**Pull request merged**